lib/hex_web/web/templates/main.html.eex

```diff
@@ -73,6 +73,7 @@
 
     <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.1/js/bootstrap.min.js"></script>
+    <script src="/static/main.js?<%= asset_id %>"></script>
```

Comment:
sorry that file was not staged,
I have updated the commit, let me know if you want the code inside the template.